src/test/scala/chisel3/internal/NameCollisionSpec.scala

```diff
@@ -10,7 +10,7 @@ class NameCollisionSpec extends ChiselFlatSpec with Utils {
 
   it should "error on duplicated names with a correct message" in {
     (the[ChiselException] thrownBy extractCause[ChiselException] {
-      ChiselStage.elaborate(
+      ChiselStage.emitCHIRRTL(
```

Comment:
wouldn't you want to keep the elaborate but just kill the return value? We don't actually care about emitChirrtl?

Reply:
I mean, we could. However, it didn't seem useful to have a public `ChiselStage$.elaborate` method which would return nothing. This could be a test-only method. However, it didn't seem worthwhile given that we could re-use `emitCHIRRTL` here.

Use of `emitCHIRRTL` does do slightly more work for situations which are not catching exceptions. Now, FIRRTL text is generated which would not be (though immediately discarded). For tests that are catching exceptions, the runtime should be the same.

Reply:
oh yeah good point that most of these tests are expecting an exception anyway